Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM docker.io/golang:1.16 as build
-ARG VERSION
+FROM docker.io/golang:1.16 as builder
 
-WORKDIR /app
+WORKDIR /workspace
 
-COPY go.mod go.sum ./
+COPY go.mod go.mod
+COPY go.sum go.sum
 RUN go mod download
 
 COPY . .
 
-RUN make test
-RUN make build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
 
 FROM gcr.io/distroless/static:nonroot
+WORKDIR /
+COPY LICENSE .
+COPY --from=builder /workspace/manager .
+USER 65532:65532
```

Comment:
I generally would prefer to have a very similar setup to K8up, i.e. let Goreleaser build the image